mappings/net/minecraft/world/entity/ai/behavior/declarative/MemoryCondition.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_brqsauyi net/minecraft/world/entity/ai/behavior/declarative/MemoryCondition
```

Comment:
this `ai/behaviour/declarative` package doesn't exist, I'd move this back into `ai/brain`